### DISCUSSION TITLE
Log4j 1.x vulnerability

### DISCUSSION THREAD

**Discussion by JU-Chang** [pietermartin/sqlg#442] [Q&A] [answered]
3  vulnerabilities (CVE-2022-23302/23305/23307)  was found recently in Log4j. Is slqg affected by that? 
Details could be found in: https://logging.apache.org/log4j/1.2/
@pietermartin @ssalevan @metlos @JPMoresmau @diegonc

**Comment by pietermartin** [comment, marked as answer]
Sqlg does not use `JMSSink` so no it is not affected by this vulnerability.

**Comment by pietermartin** [comment]
Also Sqlg does not have an internal `log4j.properties` file, it uses the one supplied by the user.

**Reply by JU-Chang** [reply]
ok.thks
